### ISSUE TITLE
Works with python, but not with javascript

### ISSUE THREAD

**Issue by dillonforrest** [scrooloose/syntastic#462]
Hi all, I found an issue which I couldn't fix myself or find online.  

I installed Syntastic with jshint using pathogen and npm respectively.  When I type "jshint" into my terminal, this is the output (which I believe should be correct):

```
Usage:
  jshint [OPTIONS] [ARGS]

Options: 
  -v, --version BOOLEAN  display package version
      --config STRING    custom config file
      --reporter STRING  custom reporter
      --jslint-reporter  use a jslint compatible xml reporter
      --checkstyle-reporter use a CheckStyle compatible xml reporter
      --show-non-errors  show additional data generated by jshint
      --extra-ext STRING comma-separated list of file extensions to use (.js 
                         is default) 
  -h, --help             Display help and usage details
```

When I save a python file with bad code, syntastic catches all the errors.  But when I save a javascript file with bad code, no errors are caught.  

I'm running OSX 10.7 with Vim 7.3.

TLDR:  python works, javascript does not, but jshint is in my path.  

How can I make Syntastic work with javascript even though it already works with python?
